### ISSUE TITLE
Problems when display and font size are turned up for accessibility

### ISSUE THREAD

**Issue by MammonLord** [Goodwy/PlayBook#62]
### Overview

Unable to complete initial setup and perform other functions while screen size and font size are increased for accessibility. 
![Screenshot_20240301-012108](https://github.com/Goodwy/PlayBook/assets/146236959/50d2bc87-a075-4fd5-bc27-d94bcea41f98)

Please consider making every page scrollable when contents overflows. 

### How to reproduce

1. Increase the font and display size maximum in system settings.
2. Open app for the first time.

### Version

3.0.0

### Android Version

33

### Device Model

moto g 5G - 2023

**Comment by Goodwy** [comment]
It's not playable on version 4+, but it's still being tested. On versions below 4, you can turn the device to landscape mode, then the buttons should be visible.
